scripts/mobile_generator.py

```diff
@@ -9,60 +9,25 @@
     '-s',
     '--source',
     dest='json_schema',
-    help='Json shema file',
+    help='Json schema file',
     required=True
 )
+parser.add_argument(
+    '-l',
+    '--language',
+    dest='output_language',
+    help='Defaults to kotlin if omitted. For Swift use -l swift',
+    required=False
```

Comment:
Maybe use `default='kotlin'` instead, but actually I see no reason to make kotlin the default output, so maybe make `-l` option required?